src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -35,16 +35,17 @@ protected function configure(): void
     {
         $this
             ->setDefinition([
-                new InputArgument('id', InputArgument::REQUIRED, 'Specific message id to remove'),
+                new InputArgument('ids', InputArgument::REQUIRED | InputArgument::IS_ARRAY, 'Specific message ids to remove'),
```

Comment:
To be consistent with the `messenger:failed:retry` I suggest to use `id` instead of `ids`. See: https://github.com/symfony/symfony/blob/e60a876201b5b306d0c81a24d9a3db997192079c/src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php#L59

Reply:
And changing to "ids" is also not backward compatible.

Reply:
you're right, i've changed the argument name